File: CPP/Flow Control/Greatest of Three.cpp
int find_greatest_number(int a, int b, int c)
{
    // Write your code here..
    if (a >= b && a >= c)
        return a;
    else if (b >= a && b >= c)
        return b;
    else if (c >= a && c >= b)
        return c;
}

File: GeeksForGeeks-Problems/JAVA/Flow Control/Greatest of Three.java

//Back-end complete function Template for Java
import java.util.*;

public class Solution {
    public static void main(String args[]) {
        // Your code here
        Scanner sc = new Scanner(System.in);
        int a = sc.nextInt();
        int b = sc.nextInt();
        int c = sc.nextInt();
        int ans = Math.max(a, (Math.max(b, c)));
        System.out.println(ans);
    }
}